.github/workflows/test.yaml

```diff
@@ -0,0 +1,71 @@
+---
```

Comment:
Should this filename (and the `name:` values inside) specifically call out that it's focused on the cockroachDB tests?

Reply:
Done, the workflow itself is now `Test CRDB`, and this name will be prepended on all jobs in it.